web/styles/compose.css

```diff
@@ -1185,10 +1185,12 @@ textarea.new_message_textarea {
     color: var(--color-text-popover-menu);
 
     .enter_sends_choice {
-        display: flex;
-        gap: 8px;
         padding: 4px 10px;
-        align-items: flex-start;
+        grid-gap: 0 8px;
+        display: grid;
+        grid-template:
+            "radio-button instructions" 1.5em
```

Comment:
Should we document where the 1.5em and 1em values come from?

Reply:
They were unfortunately a little vibes based (i.e. I fiddled with it a bit and these numbers worked well)

Reply:
That's fine...but even the vibes can be commented as such :)

Reply:
Haha sure :)

Reply:
Something like this?

```
        /* 1em and 1.5em were chosen from playing around with different values
           and picking some that looked reasonable. */
```